provision-team/svcbusclient/Program.cs

```diff
@@ -68,17 +69,12 @@ static async Task SendMessagesAsync(string recipientEmail, string messageBody)
                         { "Message", messageBody }
                     };
 
-                    var serializer = DataContractBinarySerializer<Dictionary<string, string>>.Instance;
-                    byte[] data;
+                    string data = JsonConvert.SerializeObject(message);
 
-                    using (MemoryStream stream = new MemoryStream())
-                    {
-                        serializer.WriteObject(stream, message);
-                        data = stream.ToArray();
-                    }
+                    ServiceBusMessage sendMessage = new ServiceBusMessage(data);
```

Comment:
We should use the built-in integration with System.Text.Json by leveraging the BinaryData type:
new ServiceBusMessage(new BinaryData(message)